sdk/core/azure-core-amqp/inc/azure/core/amqp/connection.hpp

```diff
@@ -268,6 +296,35 @@ namespace Azure { namespace Core { namespace Amqp { namespace _internal {
     /** @brief Destroy an AMQP connection */
     ~Connection();
 
+    /** @brief Create a session on the current Connection object.
+     *
+     * An AMQP Session provides a context for sending and receiving messages. A single connection
+     * may have multiple independent sessions active simultaneously up to the negotiated maximum
+     * channel count.
+     *
+     * @param options The options to use when creating the session.
+     * @param eventHandler The event handler for the session.
+     */
+    Session CreateSession(SessionOptions const& options = {}, SessionEvents* eventHandler = nullptr)
+        const;
+
+    /** @brief Construct a new session associated with the specified connection over the specified
+     * endpoint.
+     *
+     * @param newEndpoint - AMQP Endpoint from which to create the session.
+     * @param eventHandler - Event handler for session events.
```

Comment:
`@param options` is not documented

Reply:
I've seen CI failing due to doxygen inconsistency, I wonder if we disabled it, but that would've been helpful.

Reply:
There are [thousands of doxygen errors](https://dev.azure.com/azure-sdk/public/_build/results?buildId=2814634&view=logs&j=a7bbfb1e-9b1d-5480-b954-b42b4949e5ba&t=096ff1de-ad77-5a07-f98a-aae88a25dd53) in our code, it would be worthwhile to enable them, but not in this pull request.